content/_data/authors/sridamul.adoc

```diff
@@ -0,0 +1,11 @@
+---
+name: "Sridhar Sivakumar"
+github: sridamul
+linkedin: sridamul
+twitter: sridamul2406
+---
+
+Sridhar is an undergraduate student from India.
+He began his open-source journey in November 2023 with Jenkins.
+In 2024, he was selected for Google Summer of Code as a contributor to work on the project link:/projects/gsoc/2024/projects/using-openrewrite-recipes-for-plugin-modernization-or-automation-plugin-build-metadata-updates/[Using OpenRewrite Recipes for Plugin Modernization or Automation Plugin Build Metadata Updates].
```

Comment:
The link `link:/projects/gsoc/2024/projects/using-openrewrite-recipes-for-plugin-modernization-or-automation-plugin-build-metadata-updates/[Using OpenRewrite Recipes for Plugin Modernization or Automation Plugin Build Metadata Updates]` is not pointing to any present or the pending page. Is this intentional @sridamul?

Reply:
Yes! The link will get updated after merging #7293

Reply:
`/projects/gsoc/2024/projects/using-openrewrite-recipes-for-plugin-modernization-or-automation-plugin-build-metadata-updates/` is the path of the file that 'valentin' created in #7297